packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -49,6 +50,15 @@ class CupertinoPageScaffold extends StatelessWidget {
   /// By default uses [CupertinoColors.white] color.
   final Color backgroundColor;
 
+  /// Whether the [child] should size itself to avoid the window's bottom padding.
+  ///
+  /// For example, if there is an onscreen keyboard displayed above the
+  /// scaffold, the body can be resized to avoid overlapping the keyboard, which
+  /// prevents widgets inside the body from being obscured by the keyboard.
+  ///
+  /// Defaults to true.
+  final bool resizeToAvoidBottomPadding;
```

Comment:
Actually this is a good opportunity to make sure the API naming is current. Unfortunately the Material scaffold's name was created before we distinguished between full 'insets' where the usable view area shrinks due to things like keyboard and 'paddings' where it's obstructions from things like the home bar and notch etc.

Let's name this resizeToAvoidBottomInset